lib/cancan/controller_additions.rb

```diff
@@ -12,7 +12,7 @@ module ClassMethods
       #   end
       #
       def load_and_authorize_resource(*args)
-        cancan_resource_class.add_before_filter(self, :load_and_authorize_resource, *args)
+        cancan_resource_class.add_before_action(self, :load_and_authorize_resource, *args)
```

Comment:
before_action is not defined in rails 3.2. https://github.com/rails/rails/blob/3-2-stable/actionpack/lib/abstract_controller/callbacks.rb

I'm concerned if this passes tests. Or I might be missing something here.